configure.ac

```diff
@@ -3424,6 +3425,24 @@ if test "$wxUSE_OPENGL" = "yes" -o "$wxUSE_OPENGL" = "auto"; then
     fi
 fi
 
+dnl ---------------------------------------------------------------------------
+dnl libprocstat on FreeBSD
+dnl ---------------------------------------------------------------------------
+
+if test "$wxUSE_PROCSTAT" = "yes"; then
+    case "${host}" in
+      *-*-freebsd*)
+        AC_MSG_CHECKING([for libprocstat])
+        AC_CHECK_LIB(procstat, procstat_getpathname, [
+                        PROCSTAT_LINK=" -L/usr/lib -lprocstat"
```

Comment:
Is `-L/usr/lib` really needed? I'd expect the linked to look in this directory by default.

Reply:
When building static libs, the libs and app built and found the lib in `/usr/lib` but when building the shared libs linking failed with unable to find `libprocstat` so I added `-L/usr/lib`